### DISCUSSION TITLE
Praca domowa #1

### DISCUSSION THREAD

**Discussion by macborowy** [cloudstateu/ic-1-2022#1] [Announcements]
Hej, 

w komentarzach poniżej prosimy o wrzucenie odpowiedzi do pracy domowej, czyli linku do aplikacji frontend i API wystawionych na klastrze Kubernetes.

W razie jakichkolwiek pytań piszcie do nas mailowo: maciej.borowy@chmurowisko.pl i daniel.pisarek@chmurowisko.pl. Możecie też pisać tutaj na Github Discussions. Będziemy odpowiadać tak szybko jak to możliwe.

Powodzenia!

**Comment by apiecyk** [comment]
Cześć,
Link do mojej pracy domowej: http://20.103.137.218/
Api działa na adresie: http://20.103.138.17:8888

**Comment by Adamkopan** [comment]
Hej,
Link do mojej apki: http://20.103.246.117/
Api: http://20.103.246.136:8888/info
